.gitlab-ci.yml

```diff
@@ -245,6 +245,15 @@ build:base+async:
     COQ_EXTRA_CONF: "-native-compiler yes -coqide opt"
     COQUSERFLAGS: "-async-proofs on"
 
+build:quick:
+  <<: *build-template
+  stage: test
+  variables:
+    COQ_EXTRA_CONF: "-native-compiler yes -coqide opt"
+    COQUSERFLAGS: "-quick"
+  script:
+    - bin/coqc -boot -schedule-vio2vo 4 `find . -name \*.vio`
```

Comment:
I think this overrides the script from the build-template.

Reply:
is there a way to *append* ?

Reply:
I need the regular build + this extra command.

Reply:
I don't think so.

Reply:
I found `after_script:`

Reply:
Using after_script breaks the bot.